src/cli/remove.rs

```diff
@@ -14,6 +14,11 @@ use crate::project::manifest::FeatureName;
 use crate::{consts, project::SpecType, Project};
 
 /// Remove the dependency from the project
+///
+/// If the project manifest is a `pyproject.toml`, removing a pypi dependency with the `--pypi` flag will remove it from either
+/// - the native pyproject `project.dependencies` array or the native `project.optional-dependencies` table (if a feature is specified)
+/// - pixi `pypi-dependencies` tables of the default or a named feature (if a feature is specified)
+///
 #[derive(Debug, Default, Parser)]
```

Comment:
```suggestion
/// Remove the dependency from the project
#[derive(Debug, Default, Parser)]
#[clap(long_about = r"Remove the dependency from the project

If the project manifest is a `pyproject.toml`, removing a pypi dependency with the `--pypi` flag will remove it from either
- the native pyproject `project.dependencies` array or the native `project.optional-dependencies` table (if a feature is specified)
- pixi `pypi-dependencies` tables of the default or a named feature (if a feature is specified)
")]
```
[Clap doesn't keep the new lines](https://github.com/clap-rs/clap/issues/2389), `long_about` does. I think this would improve the UX of the `--help` command.

Reply:
I used verbatim_doc_comment to display new lines properly, for both `add` and `remove`